fbpic/particles/cuda_methods.py

```diff
@@ -609,9 +609,11 @@ def gather_field_gpu_cubic(x, y, z,
                 Fr_m += Sz[index_z]*Sr[index_r]*Er_m0[iz[index_z], ir[index_r]]
                 Ft_m += Sz[index_z]*Sr[index_r]*Et_m0[iz[index_z], ir[index_r]]
                 if Sz[index_z]*Sr[index_r] < 0:
-                    Fz_m += (-1.)*Sz[index_z]*Sr[index_r]*Ez_m0[iz[index_z], ir[index_r]]
+                    Fz_m += (-1.)*Sz[index_z]*Sr[index_r]* \
+                        Ez_m0[iz[index_z], ir[index_r]]
                 else:
-                    Fz_m += Sz[index_z]*Sr[index_r]*Ez_m0[iz[index_z], ir[index_r]]
+                    Fz_m += Sz[index_z]*Sr[index_r]* \
+                        Ez_m0[iz[index_z], ir[index_r]]
```

Comment:
The above changes are just there in order to avoid having more than 80 characters per line.